routing.md

```diff
@@ -305,12 +321,51 @@ DIDComm DID Document endpoints have the following format:
 
 **type**: MUST be `DIDComm`. 
 
-**serviceEndpoint**: MUST contain a URI for a transport specified in the [transports] section of this spec.
+**serviceEndpoint**: MUST contain a URI for a transport specified in the [transports] section of this spec, or a URI from Alternative Endpoints. Endpoints from the [transports] section SHOULD be used only for the reception of DIDComm messages.
```

Comment:
The sentence "Endpoints from the [transports] section SHOULD..." is good advice for some transports. This includes http, which is probably why it feels right to us. But if the transport = email, I'm not sure the sentence makes sense. If the transport is bluetooth, I'm not sure it does, either. I guess I'm challenging the notion that an endpoint that accepts DIDComm should only accept DIDComm. How would we feel about rewording to "It MAY be desirable to constraint endpoints from the [transports] section so that they are used only for the reception of DIDComm messages. This can be particularly helpful in cases where auto-detecting message types is inefficient or undesirable."

Reply:
BTW, I'm ambivalent about my previous comment. I'm effectively thinking out loud, not making a strong assertion.

Reply:
I like you reword.